docker/scripts/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+source /root/.bash_profile; \
+/bin/bash /cbio_scripts/addcBioAliases.sh; \
+/bin/bash /cbio_scripts/checkCustomConfig.sh; \
+$CATALINA_HOME/bin/startup.sh; \
```

Comment:
Is it possible to use an `exec` statement to run Catalina in the foreground, and get rid of the concurrently running shell process? Since Catalina is the process for which the container was started, it should ideally run as process id 1.